scripts/download_wikidata_lakes.py

```diff
@@ -1,42 +1,55 @@
 import requests
 from tqdm import tqdm
 import json
+import re
 
 url = 'https://query.wikidata.org/sparql'
 subquery = """
-SELECT ?item ?itemLabel
+SELECT ?item ?itemLabel ?altLabel
 WHERE 
 {
   ?item wdt:P31 wd:Q23397.
   ?article schema:about ?item.
+  OPTIONAL { ?item skos:altLabel ?altLabel . FILTER (lang(?altLabel) = "en") }
   SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
   FILTER (CONTAINS(STR(?item), 
 """
```

Comment:
Having a huge raw string is probably not very maintainable (hard to make changes in the future and understand what is happening). Can we add a `TODO: find a better way to generate this string more explicitly` please?

Reply:
Added TODO.